dependencies.tsv

```diff
@@ -1,5 +1,5 @@
 github.com/cheggaaa/pb	git	0817e3a1f8de9e3c78b159699b3c07d53e24a963	2017-03-23T12:38:18Z
-github.com/dgrijalva/jwt-go	git	afef698c326bfd906b11659432544e5aae441d44	2015-12-30T18:00:00Z
+github.com/dgrijalva/jwt-go	git	a539ee1a749a2b895533f979515ac7e6e0f5b650	2017-06-08T02:51:49Z
```

Comment:
You appear to be using the head version of the repo rather than the released version. I'd prefer to stick to the tagged release for version 3.0.0:
`github.com/dgrijalva/jwt-go     git     d2709f9f1f31ebcda9651b03077758c1f3a0018c        2016-06-16T19:15:56Z`